diff --git a/minimatch.js b/minimatch.js
index 731ed17c6852768f4e1636aabbf532c30dcbca5e..1aa15de4ca94ec418a282815f008fbe7b545ace4 100644
--- a/minimatch.js
+++ b/minimatch.js
@@ -16,7 +16,8 @@ minimatch.sep = path.sep
 
 const GLOBSTAR = Symbol('globstar **')
 minimatch.GLOBSTAR = GLOBSTAR
-const expand = require('brace-expansion')
+const braceExpansion = require('brace-expansion')
+const expand = braceExpansion.expand || braceExpansion
 
 const plTypes = {
   '!': { open: '(?:(?!(?:', close: '))[^/]*?)'},
